src/lib/eventHandlers.js

```diff
@@ -0,0 +1,20 @@
+export default {
+  insert : {
```

Comment:
Yah, that could work, and then I'd just do import * as eventHandlers, to be able to still iterate over the returned object. 

The only issue is if we want it to keep the names consistent, there's no way to keep using delete, because that's reserved. 

Based on that, it might be better to keep it as an object